src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -2033,6 +2033,12 @@ private List<CompletionResult> GetResultForIdentifier(CompletionContext completi
             result = CompletionCompleters.CompleteCommandArgument(completionContext);
```

Comment:
@vexx32, your last commit had 2 failures in `PowerShell-CI-windows`
Type extensible completion of type after using namespace.Command line: <using namespace System; [TextR>.IO.TextReader
```powershell
Expected $true, but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Parser\ExtensibleCompletion.Tests.ps1: line 112
112:                         $expected.Found | Should -BeTrue
```
Type extensible completion of type after using namespace.Command line: <using namespace System.IO; [TextR>.TextReader
```powershell
Expected $true, but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Parser\ExtensibleCompletion.Tests.ps1: line 112
112:                         $expected.Found | Should -BeTrue
```